catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -272,21 +283,92 @@ define([
                     data = this.model.toJSON();
                 }
                 data.name = this.getNodeName();
+                data.remote = this.isRemote();
```

Comment:
Thank you for letting me know about the `localNode` attribute. I'll implement this and get rid of some of the excessive code